pydruid/utils/filters.py

```diff
@@ -165,18 +166,24 @@ class Bound(Filter):
     :ivar bool lowerStrict: Strict lower inclusion. Initial value: False
     :ivar bool upperStrict: Strict upper inclusion. Initial value: False
     :ivar bool alphaNumeric: Numeric comparison. Initial value: False
+        NOTE: For backwards compatibility - Use "ordering" instead.
+    :ivar bool ordering: Sorting Order. Initial value: lexicographic
```

Comment:
Hi! I know this PR has already been merged, but I was just using the new `ordering` parameter (thank you @wjdecorte!) and realised there is a typo on the docstring variable type. It should be `string` instead of `boolean` (http://druid.apache.org/docs/latest/querying/filters.html). Should be a quick fix, let me know if you'd like me to raise a PR!

Reply:
Thanks.  I created a PR to fix the typo.  I changed the variable type to str.

Reply:
Cool, just saw it, thanks!